runtime/calamari/src/lib.rs

```diff
@@ -919,29 +933,13 @@ impl_runtime_apis! {
 				hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef70a98fdbe9ce6c55837576c60c7af3850").to_vec().into(),
 				// System Events
 				hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef780d41e5e16056765bc8461851072c9d7").to_vec().into(),
+				// Treasury Account
+				hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef7b99d880ec681799c0cf30e8886371da95ecffd7b6c0f78751baa9d281e0bfa3a6d6f646c70792f74727372790000000000000000000000000000000000000000").to_vec().into(),
```

Comment:
why is this change in a benchmarking PR?

Reply:
Previously, we didn't include `pallet-treasury`, so this storage was not in this white list.
But now calamari-runtime has included this pallet, https://github.com/paritytech/polkadot/blob/master/runtime/kusama/src/lib.rs#L2188